cmd/up.go

```diff
@@ -102,6 +104,7 @@ func Up() *cobra.Command {
 
 	cmd.Flags().StringVarP(&devPath, "file", "f", defaultManifest, "path to the manifest file")
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", "", "namespace where the up command is executed")
+	cmd.Flags().BoolVarP(&vscodeRemote, "vscode-remote", "", false, "configures vscode remote development")
```

Comment:
`--ssh` or `--remote` maybe?

Reply:
`--remote`? we do more than ssh, we also set capabilities, add volumes...